applications/trilinos_application/custom_utilities/trilinos_dof_updater.h

```diff
@@ -0,0 +1,275 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+
+#if !defined(KRATOS_TRILINOS_DOF_UPDATER_H_INCLUDED )
+#define  KRATOS_TRILINOS_DOF_UPDATER_H_INCLUDED
+
+
+// System includes
+#include <string>
```

Comment:
do you need those includes here? Are also there in the baseclass

Reply:
I believe these come from the template header in the core (I agree they can be removed, just saying because maybe we should fix it there too).